pkg/workceptor/workceptor.go

```diff
@@ -678,25 +688,91 @@ func (w *Workceptor) unitStatusForCFR(unitID string) (map[string]interface{}, er
 	return retMap, nil
 }
 
+func (w *Workceptor) attemptRemoteAction(remoteNodeID, remoteUnitID, remoteAction string, unit *workUnit) error {
+	timeout := time.After(300 * time.Second)
+	retryInterval := 500 * time.Millisecond
```

Comment:
Perhaps instead of a fixed interval, this should be a backoff scheme, where we increase the retry interval on failures, to some maximum limit.

Reply:
I've now implemented a backoff scheme